packages/react-components/src/components/StreamMedia.tsx

```diff
@@ -69,12 +83,19 @@ export const StreamMedia = (props: StreamMediaProps): JSX.Element => {
   }, [videoStreamElement]);
 
   return (
-    <div
-      className={mergeStyles(
-        isMirrored && pipEnabled ? invertedVideoInPipStyle(theme) : mediaContainer(theme),
-        styles?.root
+    <div className={container()}>
+      <div
+        className={mergeStyles(
+          isMirrored && pipEnabled ? invertedVideoInPipStyle(theme) : mediaContainer(theme),
+          styles?.root
+        )}
+        ref={containerEl}
+      />
+      {loadingState === 'loading' && (
+        <div className={loadingSpinnerContainer()}>
```

Comment:
Can we memoize this className `loadingSpinnerContainer()` otherwise it will return a new object everytime the component re-renders.

Reply:
Ideally initialize these styles outside the component at file level.

Reply:
note: this will be somewhat memoized as it returns a string and should always return the same string (so not as important to memoize as when we use the `styles` prop that always is a new object)